packages/frontend/src/utils/wallet.js

```diff
@@ -460,6 +460,9 @@ class Wallet {
         this.getAccountsLocalStorage();
         await this.setKey(accountId, keyPair);
         this.accounts[accountId] = true;
+
+        // TODO: figure out better way to inject reducer
+        store.addAccountReducer(accountId);
```

Comment:
@marcinbodnar please remove the `accountId` parameter here as it is no longer used

Reply:
Thanks. Done.